timeless/poster/tasks.py

```diff
@@ -4,10 +4,26 @@
 from celery import shared_task
 
 from timeless import DB
+from timeless.customers.models import Customer
 from timeless.poster.api import Authenticated, PosterAuthData, Poster
 from timeless.restaurants.models import Table
 
 
+def __poster_api():
+    auth_data = PosterAuthData(
+        application_id=os.environ.get("poster_application_id"),
+        application_secret=os.environ.get("poster_application_secret"),
+        redirect_uri=os.environ.get("poster_redirect_uri"),
+        code=os.environ.get("poster_code"),
+    )
+    auth_token = Authenticated(auth_data=auth_data).auth()
+    poster = Poster(auth_token=auth_token)
+    return poster
+
+
+__poster = __poster_api()
```

Comment:
@Vatavuk `__poster ` will be created once when module is loading. So, I think it should be placed in each sync methods.

Reply:
@vladarefiev You are right, unfortunately, the authentication is the problem since it will only authenticate once and the token will expire. I don't know why the Poster doesn't encapsulate authentication. I've raised an issue for that #390 .